phalcon/config/adapter/ini.zep

```diff
@@ -118,4 +120,47 @@ class Ini extends Config
 
 		return [key: this->_parseIniString(path, value)];
 	}
+	
+    /**
+     * @link https://github.com/ice/framework/blob/dev/ice/config/ini.zep#L46
+     * We have to cast values manually because parse_ini_file() has a poor implementation.
+     *
+     * @param mixed ini The array casted by `parse_ini_file`
+     * @return mixed
+     */
+    private function _cast(ini)
```

Comment:
1. `-> var` and remome `@return`
2. after a few months it is likely the `@link` will not be valid (at least line number)